guides/source/active_record_basics.md

```diff
@@ -179,10 +181,11 @@ end
 ```
 
 If you do so, you will have to define manually the class name that is hosting
```

Comment:
```suggestion
If you do so, you will have to manually define the class name that is hosting
```

Not part of your changes but I just saw this

Reply:
Applied in [e18a091](https://github.com/rails/rails/pull/46700/commits/e18a0911bf7551f4c940fcbb8af9d0bc36197451)